src/components/welcome/index.tsx

```diff
@@ -1,57 +1,118 @@
+import LoadSafeIcon from '@/public/images/welcome/load-safe.svg'
+import useWallet from '@/hooks/wallets/useWallet'
+import useConnectWallet from '@/components/common/ConnectWallet/useConnectWallet'
+import Keyhole from '@/components/common/icons/KeyholeIcon/keyhole.svg'
+import CircularIcon from '@/components/common/icons/CircularIcon'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
 
 const NewSafe = () => {
   const router = useRouter()
+  const wallet = useWallet()
+  const connectWallet = useConnectWallet()
 
   return (
-    <>
-      <Typography variant="h1" mb={2}>
-        Welcome to the Safe.
-      </Typography>
-      <Typography variant="h4" mb={2}>
-        Safe is the most trusted platform to manage digital assets.
-        <br />
-        Here is how to get started:
-      </Typography>
-      <Paper sx={{ padding: 3, maxWidth: '800px' }}>
-        <Grid container gap={3}>
-          <Grid item md={6}>
-            <Typography variant="h2" mb={1}>
-              Create Safe
+    <Grid container spacing={3}>
+      <Grid item xs={12} md={4} lg={3.5}>
+        <Accordion className={css.accordion} defaultExpanded={false} sx={{ display: { xs: 'block', md: 'none' } }}>
+          <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+            <Typography variant="h4" display="inline" fontWeight={700}>
+              My Safes
             </Typography>
-            <Typography mb={4}>
-              Create a new Safe that is controlled by one or multiple owners. You will be required to pay a network fee
-              for creating your new Safe.
-            </Typography>
-            <Track {...CREATE_SAFE_EVENTS.CREATE_BUTTON}>
-              <Button variant="contained" onClick={() => router.push(AppRoutes.newSafe.create)}>
-                + Create new Safe
-              </Button>
-            </Track>
-            <Divider orientation="vertical" flexItem />
-          </Grid>
-          <Grid item md>
-            <Typography variant="h2" mb={1}>
-              Load Existing Safe
-            </Typography>
-            <Typography variant="body1" mb={4}>
-              Already have a Safe or want to access it from a different device? Easily load your Safe using your Safe
-              address.
-            </Typography>
-            <Track {...LOAD_SAFE_EVENTS.LOAD_BUTTON}>
-              <Button variant="outlined" onClick={() => router.push(AppRoutes.newSafe.load)}>
-                Add existing Safe
-              </Button>
-            </Track>
-          </Grid>
-        </Grid>
-      </Paper>
-    </>
+          </AccordionSummary>
+
+          <AccordionDetails sx={{ padding: 0 }}>
+            <div className={css.sidebar}>
+              <SafeList closeDrawer={() => {}} />
+            </div>
+          </AccordionDetails>
+        </Accordion>
+        <Box className={css.sidebar} display={{ xs: 'none', md: 'block' }}>
+          <SafeList closeDrawer={() => {}} />
```

Comment:
Maybe better making the prop optional?

Reply:
Good idea, we can actually use that instead of checking for `isWelcomePage`

Reply:
I think we lose a bit of readability that way 😬 . I would just make the prop optional to not have to pass the empty function and use the other route util from `SideDrawer`.